### DISCUSSION TITLE
Allow reporting in Rank Assitant Replies mode

### DISCUSSION THREAD

**Discussion by POBIX** [LAION-AI/Open-Assistant#3454] [Ideas]
There should be a way to report harmful assistant replies. Especially when they're synthetic, I'd like to be able to take more drastic action than ranking them worst.

![image](https://user-images.githubusercontent.com/43281098/223580016-c85bf513-4972-4a8f-8412-79cf28ad57a4.png)

**Comment by olliestanley** [comment]
I think the current workflow/plan for reward model training would not be able to use the labels, we only want rankings for the synthetic data as it stands. We don't want to ask site users to do extra work if it won't be utilised. If someone from the ML team wants to correct me feel free though
